### PR TITLE
feat(yatai): Add delete button

### PR DESCRIPTION
**Pull request by jmc529**
## Description

This adds a delete button to the web UI. The button triggers a modal dialog that informs the user of their decision, then attempts to delete a Bento bundle. Upon deletion it triggers a global toast informing the user of the deletion or if an error occurred.
Here is a [link to a gif](https://imgur.com/GnDs16C) of it in action.

## Motivation and Context

This helps build out functionality for the Yatai web UI and is in response to issue #1445 

## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/yatai/web/src/index.ts [hunk 1]
```diff
@@ -8,11 +8,6 @@ const port = args[1];
 const base_log_path = args[2];
 const base_url = args[3];
 const prometheus_address = args[4];
-if (!grpcServerAddress || !prometheus_address || !port || !base_log_path) {
```
Comment (aarnphm) [inline bentoml/yatai/web/src/index.ts:11]: I think we do need this to validate the run options under `package.json` or the given commands under `Makefile`.

**Review by aarnphm — COMMENTED** [review]
@jmc529 Thank you for the PR. Overall it looks solid. We will merge this in once we finish 1.0 🎊

File: bentoml/yatai/web/src/index.ts (hunk 1 above)
Comment (jmc529) [reply bentoml/yatai/web/src/index.ts]: Thank you for pointing this out. I removed this in err.

**Comment by aarnphm** [comment]
actually @jmc529 can you create a different pr that points towards the `bentoml-1.0` branch, under the new yatai folder structure? ty

**Pull request closed without merging**